Cargo.toml

```diff
@@ -12,7 +13,7 @@ resolver = "2"
 [workspace.package]
 version = "0.1.0"
 edition = "2021"
-rust-version = "1.82.0"
+rust-version = "1.71.0"
```

Comment:
since `cosmian_findex_cli` is also re-exported in the meta cli `cosmian_cli`, I've just aligned these 3 workspaces to the same `rustc` version (actually the one of KMS).